nncf/api/compression.py

```diff
@@ -329,3 +329,22 @@ def get_transformation_layout(self, model: ModelType) -> TransformationLayout:
         :return: The instance of the `TransformationLayout` class containing
             a list of algorithm-specific modifications.
         """
+
+
+class CompressionLevel(OrderedEnum):
```

Comment:
Could you add a description that the class is deprecated and will be removed in the future release? Please, open the ticket on this task if we have not yet.